systest/backup/advanced-scenarios/127-Namespace/backup_test.go

```diff
@@ -0,0 +1,91 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+	"os"
+	"testing"
+
+	"github.com/dgraph-io/dgraph/systest/backup/common"
+	"github.com/dgraph-io/dgraph/testutil"
+)
+
+var (
+	restoreLocation = "/data/backups/"
+	backupDst       = "/data/backups/"
+	localBackupDest = "./data/backups"
+	headerAlpha1    = http.Header{}
+	headerAlpha2    = http.Header{}
+)
+
+const (
+	accessJwtHeader = "X-Dgraph-AccessToken"
+)
+
+func TestDeletedNamespaceID(t *testing.T) {
+	err := common.RemoveContentsOfPerticularDir(t, "./data/backups/")
+	if err != nil {
+		fmt.Println(err)
+		os.Exit(1)
+	}
+
+	jwtTokenAlpha1 := testutil.GrootHttpLogin("http://" + testutil.SockAddrHttp + "/admin").AccessJwt
+	headerAlpha1.Set(accessJwtHeader, jwtTokenAlpha1)
+
+	jwtTokenAlpha2 := testutil.GrootHttpLogin("http://" + testutil.ContainerAddr("alpha2", 8080) + "/admin").AccessJwt
+	headerAlpha2.Set(accessJwtHeader, jwtTokenAlpha2)
+
+	common.AddNamespaces(t, 20, headerAlpha1, "")
+
+	common.AddSchema(t, headerAlpha1, "alpha1")
+
+	common.CheckSchemaExists(t, headerAlpha1, "alpha1")
+
+	common.AddData(t, 1, 10, jwtTokenAlpha1, "alpha1")
+
```

Comment:
we don't need a new line after every line of code

Reply:
Resolved